src/utils/dev-tools/dev-tools.js

```diff
@@ -1,5 +1,7 @@
 import React from 'react'
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Since you removed all that dev. tooling from main index.js I guess you could also remove this file.